mobile/src/main/res/layout/dialog_items_unavailable_warning.xml

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
I'm fairly sure you don't actually need the frame in this case since there is nothing else in it and you're giving it another parent when you create the dialog. Mind trying to just make the `CheckBox` the only element in here?